src/parser/jobs/mnk/modules/Brotherhood.tsx

```diff
@@ -0,0 +1,64 @@
+import {t, Trans} from '@lingui/macro'
+import {DataLink} from 'components/ui/DbLink'
+import {StatusRoot} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {BuffWindow, EvaluatedAction} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {GlobalCooldown} from 'parser/core/modules/GlobalCooldown'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+import {PlayersBuffedEvaluator} from './evaluators/PlayersBuffedEvaluator'
+
+export interface BrotherhoodBuff {
+	start: number
+	target: string
```

Comment:
Might need to update imports but targets are actors.
```suggestion
	target: Actor
```